content/en/blog/_posts/2024-06-20-pod-failure-policy-ga

```diff
@@ -0,0 +1,124 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Pod failure policy goes GA"
+date: 2024-06-20
+slug: kubernetes-1-31-pod-failure-policy-stable
+author: >
+  Michał Woźniak (Google)
+---
+
+This blog discusses the Pod Failure Policy
+[Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy)
+for Jobs as it graduates to stable in 1.31.
+
+## Why to use it? {#pod-failure-policy-why-to-use}
+
+Consider running a large computational workload, comprising thousands of
```

Comment:
I think it would be better to specify the benefits of the policy for all users first (not just the ones running large workloads).

We can mention the impact on large workloads later in the text.

Reply:
Thanks for the comment. I follow this idea in the most recent version. PTAL.